front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Settings/index.tsx

```diff
@@ -58,6 +58,7 @@ const EditCustomPageSettings = () => {
         }}
         showNavBarItemTitle={hasNavbarItem}
         onSubmit={handleOnSubmit}
+        hideSlug={customPage.attributes.code === 'proposals'}
```

Comment:
```suggestion
        hideSlug={customPage.attributes.code !== 'custom'}
```

I even think it should only be editable when it's a custom page and we can get into similar trouble if we'd change other "default" pages' slugs.

Reply:
Can you check that?

Reply:
I guess- you guys worked on this last time, I don't know what kind of decisions were made around which slugs you can edit and which you can't. Happy to limit slug editing only to custom pages, but maybe that's also something that should be ran by product?

Reply:
Also, shouldn't the backend be the source of truth about which types of pages you can edit the slug of and which you can't?

Reply:
> but maybe that's something that should be discussed with product?

Yes, that is something that needs to be checked with product. That's what I was hinting at you should do. 😉 but we can already merge this and widen the hideSlug check to any non-custom page if they confirm. I'll write product.

> Also, shouldn't the backend be the source of truth about which types of pages you can edit the slug of and which you can't?

Yes, the back-end should return an error if we edit slugs that are not allowed, but that won't help decide in the FE whether we should display the slug field initially. I don't think we have a "show slug field" setting stored in the BE.